synapse/storage/roommember.py

```diff
@@ -147,7 +147,7 @@ def get_hosts_in_room(self, room_id, cache_context):
         hosts = frozenset(get_domain_from_id(user_id) for user_id in user_ids)
         defer.returnValue(hosts)
 
-    @cached(max_entries=500000, iterable=True)
+    @cached(max_entries=100000, iterable=True)
```

Comment:
Is there a reason this number is going down at the same time as changing the cache structure?